src/webui/webapplication.cpp

```diff
@@ -214,6 +214,14 @@ void WebApplication::action_public_images()
     header(Http::HEADER_CACHE_CONTROL, MAX_AGE_MONTH);
 }
 
+namespace
+{
+    inline QString toUtf8(const QByteArray& a)
```

Comment:
fromUtf8
Meaning: we create QString **from** UTF-8 encoded data.
Besides, I don't think this shortcut is so necessary.